package.json

```diff
@@ -39,10 +39,10 @@
     "typedoc": "^0.13.0",
     "typescript": "^3.1.3",
     "uuid": "^3.1.0",
-    "wdio-mocha-framework": "0.5.9",
-    "wdio-selenium-standalone-service": "0.0.8",
-    "wdio-spec-reporter": "0.1.0",
-    "webdriverio": "4.9.2"
+    "wdio-mocha-framework": "0.5.13",
```

Comment:
@benoitf Would you have an example or two of CQs for "dev-dependencies"? When I have time I'll register all those not yet covered for this repo. But maybe for now we can start registering these CQs when we add or change versions. 

Or even better, if you know the steps and are willing, you could update the [wiki](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-new-or-updated-testbuild-dependency)

Reply:
@vince-fugnitto Let's look together at registering CQs for the dev-dependencies updated in this PR. It simpler for these, since it's 1 CQ per dependency, without having to take into account their own recursive dependencies. It was mentioned before that such CQs can be requested for a version range, so hopefully asking for "current version or newer" would cover each one for a while.

Reply:
hello for dev dependencies it's one CQ with all dependencies   
  Then you specify the current version and add 'and later' so it's valid after new versions
Let me find a link of some existing requests

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14312

Reply:
Thanks for the example @benoitf . 

Question: I found this related, slightly older CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14222 . Do I understand correctly that 14222 covered most Che dev-dependencies, and that 14312 added a few, maybe that were missed or added soon after the original CQ? 

If so, it means we can regroup many such dependencies in one CQ, but if we miss some or some are later added, a new CQ only needs to cover the new dependencies, and need not be cumulative and list again those already covered?

Reply:
@marcdumais-work yes correct. As it's only subject for a vote by PMC, we can group them and we later create new bunch (previous are already covered)